soc/nxp/mcx/mcxw/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 
 zephyr_sources(soc.c mcxw71_platform_init.S)
 
+zephyr_sources_ifdef(CONFIG_NXP_NBU
+  ../../common/nxp_nbu.c
+  )
```

Comment:
not required, but if you can find a cmake variable for an absolute path to nxp/common, might be better, if not, thats fine